figures/3.figure3/scripts/3.assemble_montage.r

```diff
@@ -381,8 +379,11 @@ apoptosis_ground_truth_list <- c(ground_truth$Apoptosis$apoptosis_groups_list)
 pyroptosis_ground_truth_list <- c(ground_truth$Pyroptosis$pyroptosis_groups_list)
 control_ground_truth_list <- c(ground_truth$Healthy$healthy_groups_list)
 
-pyroptosis_ground_truth_list
 control_ground_truth_list
+apoptosis_ground_truth_list
+pyroptosis_ground_truth_list
+# replace
```

Comment:
Recommend removing any commented-out code. This looks to maybe be an unfinished code comment.